executor/aggregate_test.go

```diff
@@ -539,3 +539,17 @@ func (s *testSuite) TestIssue5663(c *C) {
 	tk.MustQuery("select group_concat(i) from t1 where i > 1;").Check(testkit.Rows("2,3"))
 	tk.MustQuery("select group_concat(i) from t1 where i > 1;").Check(testkit.Rows("2,3"))
 }
+
+func (s *testSuite) TestMaxMinFloatScalaFunc(c *C) {
+	tk := testkit.NewTestKitWithInit(c, s.store)
+	plan.GlobalPlanCache = kvcache.NewShardedLRUCache(2, 1)
```

Comment:
Is this issue related with plan-cache?

Reply:
No, this is an irrelevant change, already reverted.